plugin-modernizer-core/src/test/java/io/jenkins/tools/pluginmodernizer/core/extractor/MetadataCollectorTest.java

```diff
@@ -193,14 +194,16 @@ void testPluginWithJenkinsfileWithJdkInfo() {
         assertTrue(pluginMetadata.hasFile(ArchetypeCommonFile.JENKINSFILE));
         assertTrue(pluginMetadata.hasFile(ArchetypeCommonFile.POM));
 
-        List<JDK> jdkVersion = pluginMetadata.getJdks();
+        Set<JDK> jdkVersion = pluginMetadata.getJdks();
 
         assertEquals(2, jdkVersion.size());
         assertTrue(jdkVersion.contains(JDK.JAVA_21));
         assertTrue(jdkVersion.contains(JDK.JAVA_17));
     }
 
     @Test
+    @Disabled(
+            "This test is disabled because the collector is not able to extract the JDK version from the Jenkinsfile yet")
```

Comment:
On my opinion is better to disabled this failed tests because it's not implemented

The expected JDK is 2 not 0

Reply:
Hey @jonesbusy, I have raised #224 to resolve the issue. So, I think this is not needed anymore.